opensrp-app/src/main/java/org/smartregister/repository/TaskRepository.java

```diff
@@ -98,7 +104,10 @@
                     STRUCTURE_ID + " VARCHAR, " +
                     REASON_REFERENCE + " VARCHAR, " +
                     LOCATION + " VARCHAR, " +
-                    REQUESTER + " VARCHAR  )";
+                    REQUESTER + " VARCHAR,  " +
+                    RESTRICTION_REPEAT + " INTEGER,  " +
+                    RESTRICTION_START + " VARCHAR,  " +
```

Comment:
Change this type to INTEGER and the `restriction_end` type also. This is queried as long on line 313. Similar changes on line 131 for migration scripts. Also when saving in the ContentValue

Reply:
Changed